jsonschema-core.md

```diff
@@ -2079,7 +2079,7 @@ consumers will need to properly interpret validation results.
 ### Format
 
 JSON Schema output is defined using the JSON Schema data instance model as
-described in section 4.2.1. Implementations MAY deviate from this as supported
+described in [Instance Data Model](#data-model). Implementations MAY deviate from this as supported
```

Comment:
Do we have a way to link to another document?  I'm not particularly keen on the current method of referencing down to the reference section since it can't link directly to a section.  (I don't need this here, but I couldn't find a way to do it.)

Reply:
You can link to another document using the normal markdown link syntax. Personally, I'm not a fan of the whole linking to the references section pattern. I was going to suggest we remove that and just link to whatever we need to link to.